qdrant-landing/content/qdrant-for-startups/qdrant-for-startups-faq.md

```diff
@@ -2,23 +2,39 @@
 title: Program FAQ
 questions:
 - id: 0
-  question: Who is eligible?
+  question: What are the eligibility requirements?
   answer: |
     <ul>
       <li>Pre-seed, Seed or Series A startups (under five years old)</li>
-      <li>Has not previously participated in the Qdrant for Startups program</li>
-      <li>Must be building an AI-driven product or services (agencies or devshops are not eligible)</li>
-      <li>A live, functional website is a must for all applicants</li>
+      <li>New user of Qdrant Cloud</li>
+      <li>Not a previous participant in the Qdrant for Startups program</li>
+      <li>Be building an AI-driven product or service (agencies or devshops are not eligible)</li>
+      <li>Have a live, functional website</li>
       <li>Billing must be done directly with Qdrant (not through a marketplace)</li>
     </ul>
 - id: 1
   question: When will I get notified about my application?
   answer: Upon submitting your application, we will review it and notify you of your status within 7 business days.
 - id: 2
   question: What is the price?
-  answer: It is free to apply to the program. As part of the program, you will receive up to a 20% discount on Qdrant Cloud, valid for 12 months. For detailed cloud pricing, please visit qdrant.tech/pricing.
+  answer: It is free to apply to the program. As part of the program, you will receive a discount on Qdrant Cloud, valid for 12 months. For detailed cloud pricing, please visit qdrant.tech/pricing.
 - id: 3
   question: How can my startup join the program?
   answer: Your startup can join the program by simply submitting the application on this page. Once submitted, we will review your application and notify you of your status within 7 business days.
+- id: 4
+  question: What criteria are used to select startups for the program?
+  answer: We evaluate applications based on the innovation potential of the tech or AI-driven products or services and their alignment with Qdrant’s capabilities. Startups that demonstrate a clear vision and potential for impactful use of our platform are more likely to be selected.
+  - id: 5
```

Comment:
YAML notation requires keeping the same number of whitespaces before each list item (line starting with `-`). Now Hugo can not understand that this is the continuation of the same list and build fails. IDs 5 to 7.